debian/ibverbs-providers.symbols

```diff
@@ -98,6 +99,7 @@ libmlx5.so.1 ibverbs-providers #MINVER#
  mlx5dv_free_var@MLX5_1.12 28
  mlx5dv_pp_alloc@MLX5_1.13 29
  mlx5dv_pp_free@MLX5_1.13 29
+ mlx5dv_dr_domain_set_reclaim_device_memory@MLX5_1.14 30
```

Comment:
I think that it should be sorted.

Reply:
It's sorted by version as expected.

Reply:
It should be sorted by name and not by version.

Thanks